app/src/utility/constants.ts

```diff
@@ -1,2 +1,2 @@
-export const API_URL = process.env.REACT_APP_API_URL;
+export const API_URL = "http://localhost:9000/";
```

Comment:
NOOOOOO

Reply:
Wtfffff again???? Sorry

Reply:
You should really create a .env file in app, add:
```
REACT_APP_API_URL=http://localhost:9000/
REACT_APP_API_KEY=<Perry's Youtube Key>
```